### PR TITLE
Updating MySQL usage in README

### PR DESCRIPTION
**Pull request by mjbrender**
Walked through installation of MySQL, creation of the first database and formatted appropriately. I filled in a lot of skill gaps that I didn't have as a MySQL novice. I hope you find it helpful. Feel free to recommend edits on the PR and I'll update.

### CODE REVIEW

**Comment by andygrunwald** [comment]
This looks good. Thank you for this contribution. This might be really helpful for others.

**Comment by mjbrender** [comment]
Glad to help - thanks for the prompt merge :smile:

File: README.md [hunk 1]
```diff
@@ -46,43 +47,64 @@ Installation
 ------------
 You can install MLStats running setup.py script:
 
-    $ python setup.py install
+```python
+$ python setup.py install
+```
```
Comment (gpoo) [inline README.md]: This line is not ``python``, it is ``shell``.

File: README.md [hunk 2]
```diff
@@ -46,43 +47,64 @@ Installation
...
 
 If you don't have root privileges, use the `--prefix` option to indicate
 the directory where `mlstats` will be installed. For more details, take a
 look at the help of the installer:
 
-    $ python setup.py install --help
+```python
+$ python setup.py install --help
+```
```
Comment (gpoo) [inline README.md]: ditto

**Comment by mjbrender** [comment]
@gpoo yup, these are minor typos. Feel free to fix in a new PR given this one already merged.

**Pull request merged**